dev.py

```diff
@@ -880,24 +880,31 @@ def task_pep8diff():
     return {
         'basename': 'pep8-diff',
         'actions': [str(Dirs().root / 'tools' / 'lint_diff.py')],
-        'doc': 'Lint only files modified since last commit (stricker rules)',
+        'doc': 'Lint only files modified since last commit (stricter rules)',
     }
 
 
 @cli.cls_cmd('lint')
 class Lint():
-    """:dash: run flake8, and check PEP 8 compliance on branch diff."""
+    """:dash: check PEP 8 compliance on branch diff."""
+    def run():
+        run_doit_task({'pep8-diff': {}})
+
+
+@cli.cls_cmd('flake8')
+class Flake8():
+    """:dash: run flake8 on the scipy and benchmark directories."""
```

Comment:
While we're at it consistency-fixups-wise: could you capitalize the first letter, here and for the `build` command and the `--no-build` option?